ansible_base/lib/utils/response.py

```diff
@@ -46,13 +47,19 @@ def get_fully_qualified_url(view_name: str, *args, **kwargs) -> str:
     Returns a fully qualified URL from the setting FRONT_END_URL or a DRF reverse if the setting is undefined
     NOTE: a DRF reverse could be relative or not depending on the request coming in
     '''
-    url = drf_reverse(view_name, *args, **kwargs)
     front_end_url = get_setting('FRONT_END_URL', None)
-    if not front_end_url:
-        return url
-
-    url_pieces = urlparse(url)
-    return f"{front_end_url.rstrip('/')}{url_pieces.path}"
+    if front_end_url:
+        url = drf_reverse(view_name, *args, **kwargs)
+        url_pieces = urlparse(url)
+        return f"{front_end_url.rstrip('/')}{url_pieces.path}"
+
+    if 'request' in kwargs and kwargs['request']:
+        request = kwargs.pop('request')
+    else:
+        request = get_current_request()
```

Comment:
This will still return None if `crum.CurrentRequestUserMiddleware` isn't in middleware right?

Perhaps a log statement to this effect would be nice to add.

Reply:
There are multiple ways this can return `None`. CLI/management commands is another. I didn't log here because I'm just assuming it can return `None` in some cases as normal behavior.